<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FntSettingsPage</class>
 <widget class="QWidget" name="FntSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0,5,0,5">
   <property name="topMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="KanasMCQFontSizesContainer" stretch="1,1,1,1,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="KanasMCQTitle">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Kanas MCQ Fonts Sizes :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KanasStemSizeContainer" stretch="2,3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="KanasStemLabelContainer">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="KanasStemLabel">
           <property name="text">
            <string>Stem Size :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="KanasStemValueLabel">
           <property name="text">
            <string>20</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="KanasStemSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KanasAnswerKanaRmjSizeContainer" stretch="2,3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="KanasAnswerKanaRmjLabelContainer">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="KanasAnswerKanaRmjLabel">
           <property name="text">
            <string>Answers Size
(Kana-&gt;Rmj) : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="KanasAnswerKanaRmjValueLabel">
           <property name="text">
            <string>20</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="KanasAnswerKanaRmjSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KanasAnswerRmjKanaSizeContainer" stretch="2,3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="KanasAnswerRmjKanaLabelContainer">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="KanasAnswerRmjKanaLabel">
           <property name="text">
            <string>Answers Size
(Rmj-&gt;Kana) : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="KanasAnswerRmjKanaValueLabel">
           <property name="text">
            <string>20</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="KanasAnswerRmjKanaSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KanasAutoResizeContainer">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="KanasAutoResizeCheckbox">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Allow silent size adaptation :</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="VocabMCQFontSizesContainer" stretch="1,1,0,1,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="VocabTitle">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Vocabulary MCQ Fonts Sizes :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="VocabStemSizeContainer" stretch="2,3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="VocabStemLabelContainer">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="VocabStemLabel">
           <property name="text">
            <string>Stem Size :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VocabStemValueLabel">
           <property name="text">
            <string>20</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="VocabStemSlider">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="VocabAnswerKanaRmjSizeContainer" stretch="2,3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="VocabAnswerKanaRmjLabelContainer">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="VocabAnswerKanaRmjValueLabel">
           <property name="text">
            <string>Answers Size
(Kana-&gt;Rmj) : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VocabAnswerKanaRmjLabel">
           <property name="text">
            <string>20</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="VocabAnswerKanaRmjSlider">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="VocabAnswerRmjKanaSizeContainer" stretch="2,3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="VocabAnswerRmjKanaLabelContainer">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="VocabAnswerRmjKanaValueLabel">
           <property name="text">
            <string>Answers Size
(Rmj-&gt;Kana) : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VocabAnswerRmjKanaLabel">
           <property name="text">
            <string>20</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="VocabAnswerRmjKanaSlider">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="VocabAutoResizeContainer">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="VocabAutoResizeCheckbox">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Allow silent size adaptation :</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="FontFamiliesContainer" stretch="1,1,1,1,1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="HiraganaTitle">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Font Families :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HiraganaFtSubContainer">
       <item>
        <widget class="QLabel" name="HiraganaFntLabel">
         <property name="text">
          <string>Hiragana Font :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="HiraganaFntDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KatakanaFntSubContainer">
       <item>
        <widget class="QLabel" name="katakanaFntLabel">
         <property name="text">
          <string>Katakana Font : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="KatakanaFntDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KanjiSubContainer">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="KanjiFntLabel">
         <property name="text">
          <string>Kanji Font : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="KanjiFntDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="RomanjiFntSubContainer">
       <item>
        <widget class="QLabel" name="RomanjiFntLabel">
         <property name="text">
          <string>Romanji Font : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="RomanjiFntDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
